apps/web/src/pages/Pools/PoolAdvertisementPage/components/SkillAccordion.tsx

```diff
@@ -45,11 +45,13 @@ const Context = ({ required }: ContextProps) => {
 };
 
 interface AccordionSubtitleProps {
+  skillLevel: string | null | undefined;
```

Comment:
This is a little weird - passing in the enum in addition to the formatted string just to know whether to display the formatted string.  Maybe this first prop could be switched to a bool like "isSkillLevelAvailable" ?  Or change skillLevelItem to be string | null and just test for null before displaying it?

Reply:
I agree, it feels weird. I've taken your suggestion though and made the prop a boolean instead with [823a3a5](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10619/commits/823a3a5017aaa787da4ebf85304ee61e41c9889b). 

The problem with `skillLevelItem` is that it will never be null as it is always has a value regardless of `skillLevel`'s value is, which is really the crux of #10612.

```
  const skillLevelItem = `${`${
    intl.formatMessage({
      defaultMessage: "Level",
      id: "bVRixs",
      description: "Label displayed on the classification form level field.",
    }) + intl.formatMessage(commonMessages.dividingColon)
  } ${skillLevel}`}`;
```